### DISCUSSION TITLE
Endpoint for deleting a BASE

### DISCUSSION THREAD

**Discussion by jnsougata** [deta/cloud-docs#259] [Ideas] [answered]
Currently if you want to delete a base you have to do it manually from Base UI
It will be really helpful if there's an endpoint in `HTTP API` to delete a Base then we will be able to do it from code.
I guess there's already an endpoint beacuse of which we are able to delete base using the UI which has not been exposed to docs.

**Comment by abdelhai** [comment, marked as answer]
this is on our todos – thanks for reporting! 🙏
